File: app/src/main/java/wangdaye/com/geometricweather/main/ui/adapter/daily/holder/LineHolder.java
package wangdaye.com.geometricweather.main.ui.adapter.daily.holder;

import android.view.LayoutInflater;
import android.view.ViewGroup;

import wangdaye.com.geometricweather.R;
import wangdaye.com.geometricweather.main.ui.adapter.daily.DailyWeatherAdapter;

public class LineHolder extends DailyWeatherAdapter.ViewHolder {

    public LineHolder(ViewGroup parent) {
        super(LayoutInflater.from(parent.getContext())
                .inflate(R.layout.item_weather_daily_line, parent, false));
    }

    @Override
    public void onBindView(DailyWeatherAdapter.ViewModel model, int position) {
        // do nothing.
    }
}

File: app/src/main/java/org/breezyweather/daily/adapter/holder/LineHolder.kt
package org.breezyweather.daily.adapter.holder

import android.view.LayoutInflater
import android.view.ViewGroup
import org.breezyweather.R
import org.breezyweather.daily.adapter.DailyWeatherAdapter

class LineHolder(parent: ViewGroup) : DailyWeatherAdapter.ViewHolder(
    LayoutInflater.from(parent.context)
        .inflate(R.layout.item_weather_daily_line, parent, false)
) {
    override fun onBindView(model: DailyWeatherAdapter.ViewModel, position: Int) {
        // do nothing.
    }
}
